src/rust/engine/src/context.rs

```diff
@@ -171,10 +168,16 @@ impl Core {
       platform_properties: remote_execution_extra_platform_properties,
     };
 
+    // TODO: Propagate in an optional setting for this.
```

Comment:
Will do this before landing: it will likely default to a subdir of https://github.com/pantsbuild/pants/blob/dfd5240602c8384ac7471c1669e04dd3d337fecf/src/python/pants/base/build_environment.py#L36-L42